script.js

```diff
@@ -58,6 +58,7 @@ function removeEventListener() {
 
 function levelIncrease() {
     level++
+    document.querySelector('.level').innerHTML = 'LVL' + level
```

Comment:
no pace between `LVL` and the number, looks weird, please add a space

Reply:
it's a feature not a bug